samples/bluetooth/peripheral/src/main.c

```diff
@@ -265,6 +262,33 @@ static void bt_ready(void)
 		settings_load();
 	}
 
+	err = bt_gatt_get_svc_uuid_data(BT_DATA_UUID16_ALL, &ad[1],
+					uuid16_buf, sizeof(uuid16_buf));
+	if (err != 0) {
+		printk("Could not encode all UUID16 data: %d\n", err);
+
+		err = bt_gatt_get_svc_uuid_data(BT_DATA_UUID16_SOME, &ad[1],
+						uuid16_buf, sizeof(uuid16_buf));
+		if (err != 0) {
+			printk("Could not encode some UUID16 data: %d\n", err);
```

Comment:
"some" -> "any"?
Either able to encode all data, enable to encode some data, or not able to encode any data.
(The "some" is a bit tricky, if not able to encode all data, some data could not be encoded.  So "could not encode all" implies "could not encode some".)

Also elsewhere.

Reply:
I agree with your statement :) The reason behind using "some" here was that it was attempting to encode "some" (BT_DATA_UUID16_SOME) UUIDs. 

I can change it if you still feel like "any" is better

Reply:
Changed to `Could not encode UUID16 data`, as we at this point won't have any encoded data.